index.html

```diff
@@ -1,29 +1,233 @@
+                        <td>10</td>
+                        <td><img class="sponsors" src="Luminosity.png" alt="sponsorship company"/></td>
+                        <td>Maister</td>
+                        <td><img class="mains" src="MrGame&Watch.png" alt="main character used in tournament"/></td>
+                        <td><img class="flags" src="Mexico flag.png" alt="country of origin"/></td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+                <a class="button" href="#">View more</a>
+              </div>
+            </div>
+          </div>
+      </section>
+
+      <section>
+        <div class="tournaments">
+          <div class="container--small">
+            <h2>Recent Tournaments</h2>
+            <h3>See more</h3>
+
+            <ul class="tournament--results">
+              <li>
+                <img class="tournament--result--images" src="Elite Monthly.png" alt="Logo of tournament"/>
+                <h4>Elite Monthly Series V : Road To GOM</h4>
+                <p>February 25th, 2023</p>
+                <dl><p>Montréal, QC</p></dl>
+                <dl><p>70 Attendees</p></dl>
+              </li>
+              <li>
+                <img class="tournament--result--images" src="LVL UP EXPO.png" alt="Logo of tournament"/>
+                <h4>LVL UP EXPO 2023</h4>
+                <p>February 17th-19th, 2023</p>
+                <dl><p>Las Vegas, NV</p></dl>
+                <dl><p>739 Attendees</p></dl>
+              </li>
+              <li>
+                <img class="tournament--result--images" src="Coinbox.png" alt="Logo of tournament"/>
+                <h4>The Coinbox #49 (Ultimate)</h4>
+                <p>February 15th, 2023</p>
+                <dl><p>Online</p></dl>
+                <dl><p>783</p></dl>
+              </li>
+              <li>
+                <img class="tournament--result--images" src="Kagaribi.png" alt="Logo of tournament"/>
+                <h4>Kagaribi 9</h4>
+                <p>February 4th-5th, 2023</p>
+                <dl><p>Tokyo, JP</p></dl>
+                <dl><p>1,870 Attendees</p></dl>
+              </li>
+            </ul>
+
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <section class="contact--us">
+      <div class="form--grid">
+        <h2>What should we add?</h2>
+        <div class="contact--grid">
+          <div class="part1">
+            <ul>
+              <li><input type="text" placeholder="Full Name"></li>
+              <li><input type="text" placeholder="Email"></li>
+              <li><input type="text" placeholder="Type your message here"></li>
+            </ul>
+          </div>
+            <div class="buttoncontainer">
+              <button class="button" type="submit">Submit</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+<footer>
+    <img class="footer--logo" src="logo.png" alt="Logo of game"/>
+    
```

Comment:
The `section` above isn't in `main` or in `footer`. It's kind of just dangling in the middle of no where. 

Would recommend moving that section's content into either main or footer.